tests/display.rs

```diff
@@ -339,4 +339,60 @@ mod generic {
             assert_eq!(s.to_string(), "10");
         }
     }
+
+    mod bound {
+        use super::*;
+
+        #[test]
+        fn bound_simple() {
```

Comment:
No need to name functions with `bound_` prefix if they are placed inside `bound` module already.